timeago/lib/src/messages/ku_messages.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:timeago/src/messages/lookupmessages.dart';
+
+class EnMessages implements LookupMessages {
```

Comment:
These classes should be named after the locale name in this case should be `KuMessages` and `KuShortMessages`